azure-iot-device/azure/iot/device/constant.py

```diff
@@ -13,6 +13,7 @@
 PROVISIONING_API_VERSION = "2019-03-31"
 SECURITY_MESSAGE_INTERFACE_ID = "urn:azureiot:Security:SecurityAgent:1"
 TELEMETRY_MESSAGE_SIZE_LIMIT = 262144
+LOAD_BALANCER_LIMIT_SECS = 1740
```

Comment:
I assume this is some max keepalive value.  In that case, I would recommend calling it MAX_KEEPALIVE and then put a comment that says the value comes from the limitations of the load balancer.

Reply:
+1

---
In reply to: [461195404](https://github.com/Azure/azure-iot-sdk-python/pull/635#discussion_r461195404) [](ancestors = 461195404)